File: count-characters/countCharacters.js
const count = (string) => {

    const output = {};

    string.split('').forEach((element) => {
        if(output[element] === undefined) {
            output[element] = 1;
        } else {
            output[element] += 1;
        };
    });

    return output;

};

module.exports = { count };


//https://www.codewars.com/kata/52efefcbcdf57161d4000091/train/javascript

File: python-venv/count_characters/kata_function/count_characters.py
def count(string):

    output = {}
    for i in range(len(string)):
        if string[i] in output:
            output[string[i]] += 1
        else:
            output[string[i]] = 1

    return output

#https://www.codewars.com/kata/52efefcbcdf57161d4000091/train/python